src/docs/asciidoc/distributed_sql.adoc

```diff
@@ -0,0 +1,211 @@
+
+
+== Distributed SQL
```

Comment:
Why don't we just name it SQL? Just a thought?

Reply:
Removed "distributed"